custom_dc/manage_repo.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: default
+title: Manage a repo
+nav_order: 5
+parent: Custom Data Commons
+---
+
+{:.no_toc}
+# Manage a repo
+
+* TOC
+{:toc}
+
+
+## Use a prebuilt image
+
+While you are just testing out data changes, you don't need to build the website, but can just use a prebuilt Data Commons image.
+
+Data Commons provides two prebuilt images in the Google Artifact Registry that you can download to run in a Docker container:
+
+- `gcr.io/datcom-ci/datacommons-website-compose:stable`. This is a tested, stable version but may be several weeks old.
+- `gcr.io/datcom-ci/datacommons-website-compose:latest`. This is the latest built version that is running in production.
```

Comment:
> running in production

That is not true. It just is the latest that was built from head.

Reply:
What?  Really? So it is actually less stable than customdc_release?

Reply:
Also, what is the difference between the stable Docker image vs. the customdc_stable release then?

Reply:
The stable docker image will be based off of the customdc_stable release. We'll make it part of the oncall in the near future.

Reply:
Ah, OK. Rewritten, PTAL

Reply:
LG.